### PR TITLE
feat: minimal escape hatches for PartyBid v1.1

### PR DESCRIPTION
**Pull request by anna-carroll**

### CODE REVIEW

File: contracts/PartyBid.sol [hunk 1]
```diff
@@ -346,27 +346,51 @@ contract PartyBid is ReentrancyGuardUpgradeable, ERC721HolderUpgradeable {
...
-     * (e.g. because of a faulty MarketWrapper that marks the auction Lost)
-     * the PartyDAO Multisig can transfer the NFT to the multisig
+     * @notice Escape hatch: in case of emergency,
+     * PartyDAO can use emergencyWithdrawEth to withdraw
+     * ETH stuck in the contract
      */
-    function recover() external {
+    function emergencyWithdrawEth(uint256 _value) external {
         require(
             msg.sender == partyDAOMultisig,
```
Comment (steveklebanoff) [inline contracts/PartyBid.sol:358]: not a big deal but would be nice to have an `onlyMultisig` modifier and use it here and `emergencyForceLost`

File: test/custom/Emergency.test.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    it('Can withdraw ETH', async () => {
+        await signers[0].sendTransaction({
+            to: partyBid.address,
+            value: eth(1)
+        });
+
+        await expect(emergencyWithdrawEth(partyBid, signers[0], eth(1))).to.not.be.reverted;
+    });
+
+    it('Can force lost', async () => {
```
Comment (steveklebanoff) [inline test/custom/Emergency.test.js:54]: i'm sure you're on it, but def want tests to ensure only the multisig can call the emergency function

**Review by steveklebanoff — APPROVED** [review]
LGTM 👍

File: test/custom/Emergency.test.js (hunk 2 above)
Comment (anna-carroll) [reply test/custom/Emergency.test.js:54]: yep absolutely :)

**Pull request merged**